resolwe/flow/models/data.py

```diff
@@ -10,7 +10,8 @@
 from django.db import models, transaction
 
 from resolwe.flow.expression_engines.exceptions import EvaluationError
-from resolwe.flow.utils import dict_dot, get_data_checksum, iterate_fields, iterate_schema
+from resolwe.flow.models.utils import fill_with_defaults
```

Comment:
This commit includes several changes:

- `ResolweSlugRelatedField`
- `fill_with_defaults`
- `perform_get_or_create`

Please split them in separate commits.

Reply:
Done.